src/main/scala/rocket/RocketCore.scala

```diff
@@ -322,11 +333,45 @@ class Rocket(tile: RocketTile)(implicit p: Parameters) extends CoreModule()(p)
   val id_csr_ren = id_ctrl.csr.isOneOf(CSR.S, CSR.C) && id_expanded_inst(0).rs1 === 0.U
   val id_csr = Mux(id_system_insn && id_ctrl.mem, CSR.N, Mux(id_csr_ren, CSR.R, id_ctrl.csr))
   val id_csr_flush = id_system_insn || (id_csr_en && !id_csr_ren && csr.io.decode(0).write_flush)
+  val id_set_vconfig = Seq(Instructions.VSETVLI, Instructions.VSETIVLI, Instructions.VSETVL).map(_ === id_inst(0)).orR && usingVector.B
```

Comment:
Why not just decode it;p

Reply:
I wanted to avoid adding new column to the giant ugly decode table :/ . Do you think I should just add the new field there?

Reply:
Understand, we should refactor or rewrite the decoder in the future...